Sources/Cache/APIs/MemcachedImplementation.php

```diff
@@ -49,7 +49,10 @@ function($server)
 					return array($server, 0);
 
 				else
-					return array($server, isset($server[1]) ? (int) $server[1] : 11211);
+				{
+					$server = explode(':', $server);
+					return array($server[0], isset($server[1]) ? (int) $server[1] : 11211);
```

Comment:
didn't looked at the other pr,
but we expect that the memcached is connect by ipv4 right?

Reply:
Right